components/rhs_thread/rhs_thread.tsx

```diff
@@ -25,6 +27,15 @@ const RhsThread = ({
     channel,
     previousRhsState,
 }: Props) => {
+    const dispatch = useDispatch();
+    const currentTeam = useSelector(getCurrentTeam);
```

Comment:
do we need this? since we add `currentTeam` in `mapStateToProps`

Reply:
nope, thanks for noticing that dupe/unnecessary selector!